examples/secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request/1-Convert-Response-Claim-200_OK.xml

```diff
@@ -71,7 +71,7 @@
                 <code code="R8000" codeSystem="1.2.826.0.1285.0.2.1.104"/>
                 <telecom use="WP" value="tel:0532567890"/>
                 <agentPerson classCode="PSN" determinerCode="INSTANCE">
-                  <id extension="7654321" root="1.2.826.0.1285.0.2.1.54"/>
+                  <id extension="3415870201" root="1.2.826.0.1285.0.2.0.65"/>
```

Comment:
bit worried about this one - i THINK the requirements ask us to populate OID ending with `1.54`, just the value should be an SDS ID. think we should hold off on this until we get this clarified - probs by ameya

Reply:
Confirmed with Ameya (who confirmed with Rob Gooch) it does infact need to be the professional code root (1.54) with the SDS ID value.